test_launch_ros/test/test_launch_ros/actions/example_parameters_1.yaml

```diff
@@ -0,0 +1,12 @@
+/**:
```

Comment:
nit^N: Now we have a file named, `example_parameters.yaml` and another one named `example_parameters_1.yaml` in the same folder.

I would rather use the same style in both, i.e. `example_parameters_0.yaml` and `example_parameters_1.yaml` (_1 and _2 is also fine).